src/testdir/test_quickfix.vim

```diff
@@ -6578,4 +6578,37 @@ func Test_cbuffer_range()
   call XbufferTests_range('l')
 endfunc
 
+" Test for displaying fname pass from setqflist when the name
+" are hard links to prevent seemly duplicate entries.
+func Xtest_hardlink_fname(cchar)
+  CheckUnix
```

Comment:
It may be better to move this line to the start of `Test_hardlink_fname()`

Reply:
updated, thanks